contrib/forms/ippf_srh/new.php

```diff
@@ -124,8 +126,6 @@ function end_group()
 
 </style>
 
-<script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
```

Comment:
Testing: Dunno why this was removed

Reply:
This is included in all calls to  Header::setupHeader